CondTools/RunInfo/src/LHCInfoPopConSourceHandler.cc

```diff
@@ -340,7 +340,7 @@ bool LHCInfoPopConSourceHandler::getCTTPSData(cond::persistency::Session& sessio
   float crossingAngle = 0., betastar = 0.;
 
   bool ret = false;
-  LHCInfoImpl::LumiSectionFilter filter(m_tmpBuffer);
+  theLHCInfoImpl::LumiSectionFilter filter(m_tmpBuffer);
```

Comment:
Why not also here?
```suggestion
    LumiSectionFilter<LHCInfo> filter(m_tmpBuffer);
```

Reply:
@perrotta my bad: I suggested Jan not to touch ecessively the "old" LHCInfoPopCon because I just wanted to have this bugfix in asap. But given LHC status we have no rush, thus we can do this change also here.

@JanChyczynski can you please use the external `LumiSectionFilter` also here?